pkg/client/unversioned/discovery_client.go

```diff
@@ -154,7 +126,9 @@ func (d *DiscoveryClient) ServerResources() (map[string]*unversioned.APIResource
 
 func setDiscoveryDefaults(config *Config) error {
 	config.Prefix = ""
-	config.Version = ""
+	config.Version = "unversioned"
```

Comment:
That seems not good. Setting bogus values just to pass checks in other parts of the code…